examples/config-builder/three_qb_system.py

```diff
@@ -0,0 +1,317 @@
+                -0.06098227204421411,
+                0.36957216345202887,
+                0.29532398495147905,
+                0.26477074638298104,
+                0.3062366161339038,
+                0.22502162570719328,
+                0.1967487786023379,
+                0.18801232158046996,
+                0.17578459439160232,
+                0.10929652583018863,
+                0.0434285767839821,
+                0.030467828001564386,
+                0.02611384802414898,
+                0.01982699565942382,
+                0.012127833300662735,
+            ]
+        )
+    ).tolist(),
+)
+cnot_Q1 = ArbitraryWaveform(
+    "cnot_Q1",
+    (
+        -2.0
+        * np.pi
+        * np.array(
+            [
+                -9.897728585998266e-5,
+                0.008034988293530495,
+                -0.13374179184241197,
+                -0.19647084682136146,
+                -0.35146294139611556,
+                -0.2528581565333187,
+                -0.1250757901191075,
+                -0.08132008047619026,
+                -0.06529351116426077,
+                -0.06738991440713618,
+                -0.060797866422997615,
+                -0.026609361647279173,
+                -0.012496635056721881,
+                -0.005936085984044713,
+                -0.0021843702393362645,
+                -0.00042162333083034455,
+            ]
+        )
+    ).tolist(),
+)
+
+cnot_I2 = ArbitraryWaveform(
+    "cnot_I2",
+    (
+        2.0
+        * np.pi
+        * np.array(
+            [
+                -0.006857635028247913,
+                0.21253808896116097,
+                -0.020088639336504545,
+                -0.12584578959791412,
+                -0.11347780637171505,
+                -0.11644741135448118,
+                -0.0823775265689421,
+                -0.05812226313850513,
+                -0.05239180082948136,
+                -0.07733406271074851,
+                -0.12683149973163313,
+                -0.11575712587511292,
+                -0.06723466515593918,
+                -0.030842674949357687,
+                -0.008387866578188683,
+                0.006500701318770028,
+            ]
+        )
+    ).tolist(),
+)
+cnot_Q2 = ArbitraryWaveform(
+    "cnot_Q2",
+    (
+        -2.0
+        * np.pi
+        * np.array(
+            [
+                -0.00021459250417936717,
+                -0.08378057074732273,
+                -0.15420784656242895,
+                0.020025773137018886,
+                0.03817220075116107,
+                0.0783367771695887,
+                0.14219130502814858,
+                0.16903873908509154,
+                0.1808110064928055,
+                0.17371305493458056,
+                0.11472706694365807,
+                0.045723392279059444,
+                0.02312523234819297,
+                0.009823882427570495,
+                0.0019644568243603654,
+                -0.0008139980959453693,
+            ]
+        )
+    ).tolist(),
+)
+
```

Comment:
The ordering here is different compared to the previous example...
I actually think this makes a bit more sense, but again, add a few comments to explain it.